ecosystem/typescript/sdk/src/plugins/aptos_token.ts

```diff
@@ -130,7 +130,7 @@ export class AptosToken {
         options?.tokensBurnableByCreator ?? true,
         options?.tokensFreezableByCreator ?? true,
         options?.royaltyNumerator ?? 0,
-        options?.royaltyDenominator ?? 0,
+        options?.royaltyDenominator ?? 1,
```

Comment:
1 let's you get past with 0 for default, I guess you have to change the denominator otherwise, it'll get blocked by limits on royalties I'm assuming, 100% isn't allowed?

Reply:
https://aptos-org.slack.com/archives/C03N83P7QUC/p1690474951903069?thread_ts=1690474688.293299&cid=C03N83P7QUC